README.md

```diff
@@ -1,3 +1,6 @@
-Data Platform
+# Data Platform
 
 A repository to collect all of the helpers of Data Platform team at Canonical.
+
+A list of all the active repositories can be found using the topic '[data-platform](
+https://github.com/search?q=topic%3Adata-platform+org%3Acanonical+fork%3Atrue+archived%3Afalse&type=repositories)'.
```

Comment:
maybe better link 
```suggestion
https://github.com/orgs/canonical/teams/data-platform-engineers/repositories)'.
```

Reply:
I have almost committed it... but tested in incognito window and the link cannot be publicly used (the repo is public).
It is a nice like to use internally, but maybe we should stay with "topics" link for public usage?

Reply:
Discussed with Mykola and agree to resolve this. Merging...